.github/workflows/update-version.yml

```
@@ -0,0 +1,48 @@
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 22.x
+
+      - name: Update WhatsApp Web Version
+        id: update_version
+        run: npm run update:version
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v5
```

actionlint (1.7.9)

the runner of "peter-evans/create-pull-request@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)